cmake/compiler/icx/target.cmake

```diff
@@ -45,18 +45,29 @@ else()
   list(APPEND TOOLCHAIN_C_FLAGS "-m32")
 endif()
 
+# Search for filename in default compiler library path
+
+function(library_file_path filename i)
```

Comment:
when a function is required outside the file defining it then we should ensure both that the function is always defined, but also ensure it's properly documented.

Please find a better self-explaining name for the output var `i`.

Reply:
Maybe provide a fallback definition which does the gcc/clang thing somewhere common and then allow compiler configurations to override that?

Reply:
(I used 'i' because that was what another neighboring function used. Will fix)

Reply:
The current version provides a gcc/clang default version in compiler/target_template.cmake